packages/react-native-windows-init/src/Cli.ts

```diff
@@ -48,7 +48,19 @@ const argv = yargs.version(false).options({
   experimentalNugetDependency: {
     type: 'boolean',
     describe:
-      'Experimental change to start consuming a nuget containing a pre-built dll version of Microsoft.ReactNative',
+      '[Experimental] change to start consuming a nuget containing a pre-built dll version of Microsoft.ReactNative',
+    hidden: true,
+  },
+  nugetTestVersion: {
+    type: 'string',
+    describe:
+      '[internalTesting] By default the nuget version matches the rnw package. This flag allows manually specifying the version for internal testing.',
+    hidden: true,
+  },
+  nugetTestFeed: {
+    type: 'string',
+    describe:
+      '[internalTesting] Allows a test feed to be added to the generated nuget configuration',
     hidden: true,
   },
   useWinUI3: {
```

Comment:
useWinUI3 requires building from source right now, it's not compatible with nuget right now. Mind erroring out so people don't get the options wrong?

Reply:
This is the case for C++ too, will do in separate PR